Riot/Modules/Room/RoomViewController.swift

```diff
@@ -154,6 +154,53 @@ extension RoomViewController {
         RiotSettings.shared.enableWysiwygTextFormatting.toggle()
         wysiwygInputToolbar?.textFormattingEnabled.toggle()
     }
+    
+    @objc func didChangeMaximisedState(_ state: Bool) {
```

Comment:
Should the parameter be called `isMaximised` rather ?

Reply:
yeah makes more sense

Reply:
done